pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi, pkgs }:
```

Comment:
We don't tend to take `pkgs` in as an argument. It might be best to accept `python3Packages`, from which you can obtain `python3Packages.buildPythonApplication` and the python packages you need.

Reply:
fixed.

Reply:
have a small query: should  i append the requested changes to this init commit or push a new commit ??

Reply:
Squash it all down.